test/unit/controllers/declaration/AdditionalFiscalReferencesControllerSpec.scala

```diff
@@ -200,21 +203,22 @@ class AdditionalFiscalReferencesControllerSpec extends ControllerSpec {
 
       "user correctly add new item" in new SetUp {
 
-        val correctForm = Seq(("country", "PL"), ("reference", "12345"), addActionUrlEncoded)
+        val correctForm: Seq[(String, String)] = Seq(("country", "PL"), ("reference", "12345"), addActionUrlEncoded)
 
-        val result = controller.saveReferences("itemId")(postRequestAsFormUrlEncoded(correctForm: _*))
+        val result: Future[Result] = controller.saveReferences("itemId")(postRequestAsFormUrlEncoded(correctForm: _*))
 
         status(result) must be(SEE_OTHER)
       }
 
       "user save correct data" in new SetUp {
 
-        val correctForm = Seq(("country", "PL"), ("reference", "12345"), saveAndContinueActionUrlEncoded)
+        val correctForm: Seq[(String, String)] =
+          Seq(("country", "PL"), ("reference", "12345"), saveAndContinueActionUrlEncoded)
 
-        val result = controller.saveReferences("itemId")(postRequestAsFormUrlEncoded(correctForm: _*))
+        val result: Future[Result] = controller.saveReferences("itemId")(postRequestAsFormUrlEncoded(correctForm: _*))
 
         status(result) must be(SEE_OTHER)
-        redirectLocation(result) must be(Some("/customs-declare-exports/declaration/items/itemId/item-type"))
+        redirectLocation(of = result) must be(Some(routes.ItemTypeController.displayPage("itemId").url))
```

Comment:
The same here, what's this?

Reply:
me being stupid with intellij and trying to reduce warnings.